### DISCUSSION TITLE
Horizontal rule vs. simple table:  Is this expected behavior?

### DISCUSSION THREAD

**Discussion by bartonc-javs** [jgm/pandoc#9685] [Q&A] [answered]
I was working on a document that used horizontal rules; i.e. `- - -` at the beginning of the line.

If I put a newline after the first horizontal rule, everything rendered correctly:

    - - -
    
    ## ASDF ASDF
    
    - - -

... but I accidentally left this off:

    - - -    
    ## ASDF ASDF
    
    - - -

... and it suddenly rendered as a very weirdly formatted table.

The thing that made it extra confusing was that I had an unordered list with GFM checkboxes. If there was more than one level in the checklist, it rendered as an unordered list

    - - -    
    ## ASDF ASDF
    
    * [ ] This doesn't render as a table
        * [ ] qua qua qua 
    - - -

But when I removed the second level of list items, it turned into a table

    - - -    
    ## ASDF ASDF
    
    * [ ] This renders as a table
    - - -

This made it feel like a problem with the ordered list rather than a horizontal rule not followed by an empty line. 

I'm trying to decide whether to open a bug report on this. It may conform to a design spec, but it certainly didn't feel like a common sense rendering when it happened (especially because I usually use pipe tables)

**Comment by jgm** [comment, marked as answer]
It's technically a (malformed) simple table (see the docs).
(Of course you can turn these off if you don't use them; indeed, with `-f gfm` they will be turned off.)

**Reply by bartonc-javs** [reply]
Perfect, thanks!
